data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+	source
+		government "Coalition"
+	destination "Zug"
+	to offer
+		has "event: coalition citizens discover fangvin"
+	on offer
+		conversation
+			`Upon landing in the spaceport, you get a message from the same group of Coalition citizens. You recall that you had provided them with a datachip containing the entire knowledge of humanity.`
+			``
+			`		Dear <first>,`
+			`	Interested in an author from the datachip, we are. 'Anna Fangvin', her name is. The story about your race's actions against the oppressors, captivate us it did. To converse with her, we wish.`
+			choice
+				`	(Disregard.)`
+					decline
+				`	(Look for Anna Fangvin.)`
+			`A quick look through your encyclopedia reveals Anna Fangvin to be an author from <destination>. You decide to meet her there for an interview and set a course.`
+	on complete
+		conversation
+			`As you prepare to land, you write Anna a short message, offering an interview. In less than 10 minutes you receive a reply, explaining that Anna has accepted your offer and will wait for you in the spaceport.`
+			`	Look for Anna in the spaceport, so you can take her to the Coalition for a little conversation with those Coalition citizens who showed interest in her work.`
+
+mission "Fangvin: Struggling Author 1"
+	name "Take Anna to Coalition Space"
+	description "Take Anna Fangvin to <destination>, where she might find success as an author after having struggled on <origin> for so long."
+	source "Zug"
+	destination "Ki Patek Ka"
+	to offer
+		has "Fangvin: Book Discovered: done"
+	on offer
+		conversation
+			`Upon entering the spaceport, you walk past a restaurant. A woman then raises her arms, waving towards you. She must be Anna Fangvin, that author the Coalition citizens showed interest in.`
+			choice
+				`	(Approach her.)`
+			`	You approach the woman and sit right next to her. "Hello, are you <first>?`
+			choice
+				`	"Yes, I am. So, what do you want to ask?"`
+			choice
+				`	"What do you write?"`
+					goto "what do you write"
+			label "what do you write"
+			`	"I write science fiction novels. My latest novel was 'Beyond the Hyperdrive', which I released a few weeks ago. It's about a bunch of humans who steal jump drives from the Quarg. Here, have a copy."`
+			`	Anna retrieves a hardcover book from her bag. It has an image of a Jump Drive on it. You take the book.`
+			`	"You'll like it, I promise."`
+			choice
+				`	"How long is the book?`
+					goto "how long is the book"
+				`	"Why is it a story about humanity stealing from the Quarg?"`
+					goto "anna hates quarg"
+			label "how long is the book"
+			`	It isn't very long, it should be three hundred pages.`
+			label "anna hates quarg"
+			`	"I have a lot of problems with them. I just don't trust them. I remember when my parents took me to Humanika. Naturally, I wanted to explore, so I wandered off. I was lost. This creepy-looking Quarg found me, and when it tried to approach, I cried."`
```

Comment:
This'd have been a traumatic event for Anna, so her explaining it needs to be drawn out more. Currently this is just getting to the point, so I'd first increase the lines so that she's giving more detail, particularly on the process of how she got lost and then later was reunited with her parents (likely having to be brought to them by a Quarg, or having to wait with Quarg until they found her). Then, break up the spoken bits with narration of how she's behaving in terms of physical language. The line probably wouldn't begin with her speaking, rather with narration noting that she fidgets in her seat, or maybe looks down or at the book. Try to further break up the story to keep certain sections contained, as if she's preparing to recall and tell the next, and the next and the next (ie: admitting she's got a bunch of problems with them / saying she visited Humanika once with her parents when she was a kid / her reaction to the world and desire to explore / her getting lost, and seeing only the Quarg around). I'd also remove the emphasis on a specific "creepy" Quarg, and rather have it be the case she was more frightened due to the alien locale, the fact she winded up somewhere with no humans in sight, and only the very tall, inscrutable Quarg about. Also, maybe instead of her crying as result of *her* trying to approach a Quarg, it's due to a Quarg approaching her and trying to pick her up maybe, to get her back out of wherever she ended up in.